tools/accuracy_checker/accuracy_checker/annotation_converters/__init__.py

```diff
@@ -92,5 +93,6 @@
     'CVATPersonDetectionActionRecognitionConverter',
     'SQUADConverter',
     'XNLIDatasetConverter',
+    'CmuPanopticKeypointsConverter'
```

Comment:
```suggestion
    'CmuPanopticKeypointsConverter',
```

Reply:
@Daniil-Osokin, it should fix problem with test failed

Reply:
I get it, this happened due to merge. So will fix it in separate commit, ok.